x-pack/legacy/plugins/maps/public/layers/styles/vector/components/color/mb_validated_color_picker.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component } from 'react';
+import { isValidHex, EuiColorPicker, EuiFormControlLayoutProps } from '@elastic/eui';
+
+export const RGBA_0000 = 'rgba(0,0,0,0)';
```

Comment:
It looks like we also have this variable in `x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_color_property.js`. Maybe we should add it to `x-pack/plugins/maps/common/constants.ts` and import from there?